config/src/config/quorum_store_config.rs

```diff
@@ -107,6 +108,7 @@ impl Default for QuorumStoreConfig {
             // number of batch coordinators to handle QS batch messages, should be >= 1
             num_workers_for_remote_batches: 10,
             batch_buckets: DEFAULT_BUCKETS.to_vec(),
+            allow_batches_without_pos_in_proposal: true,
```

Comment:
The default should be set to false right to maintain backward compatibility? Actually can you ensure all unit tests pass with this being default to true (so that the UTs exercise the newly added code path) but ensure to set this to false before landing?

Reply:
Yes. I will flip the flag before landing.